### PR TITLE
delete button working

### PR DESCRIPTION
**Pull request by americool**
**- Summary**

Added the basic functionality (via a button) to delete a card from the editorial workflow. It also closes the related PR via GitHub API. It mirrors the request publish functionality. This was created to address issue #252.

**- Test plan**

Added cms to an existing statically generated site. Deleted(which closed related PRs) cards using the new button in all columns of the editorial workflow. (Note this could not be tested in the local example as discussed in issue #264.) Also checked that the cards remained deletable without error if the corresponding PR had been closed independently on github prior to use of the delete button. Attached gifs below: 

First gif demonstrates deleting cards from all rows and resulting PRs closed: 

delete_button_works

Second gif demonstrates a previously closed PR in which the card still deletes without error: 

card_deletes_when_pr_closed

**- Description for the changelog**

Adds a delete button to the editorial workflow. 

**- A picture of a cute animal (not mandatory but encouraged)**

*My dog wearing antlers for xmas* 

15590495_10101285005680159_7556777379988619321_n

### CODE REVIEW

**Comment by biilmann** [comment]
Looking good - there's a console log in there and as @bdougie mentioned we should remove the editorial_worklow setting from the example config until we have it supported in the test backend

**Comment by americool** [comment]
Thanks for the feedback, removed those and other unnecessary comments and I'm currently updating the description, should be done soon.

File: src/containers/editorialWorkflow/UnpublishedEntriesPanel.js [hunk 1]
```diff
@@ -26,14 +27,17 @@ class unpublishedEntriesPanel extends Component {
   }
 
   render() {
-    const { isEditorialWorkflow, isFetching, unpublishedEntries, updateUnpublishedEntryStatus, publishUnpublishedEntry } = this.props;
+    console.log(this.props);
```
Comment (bdougie) [inline src/containers/editorialWorkflow/UnpublishedEntriesPanel.js]: rogue console.log should be removed as well 😀

**Comment by americool** [comment]
Think I've made the necessary changes. Let me know if everything looks okay. Thanks for all the feedback!

**Comment by calavera** [comment]
Thanks a lot @americool. LGTM

**Pull request merged**